app/interfaces/api/entities/ccr_claim.rb

```diff
@@ -35,12 +44,24 @@ def defendants_with_main_first
         object.defendants.order(created_at: :asc)
       end
 
+      def dummy_retrial_reduction
+        true
```

Comment:
A little worried about exposing a dummy field... 
I take it you have a plan to remove this?